client/sites-dashboard/components/sites-table-row.tsx

```diff
@@ -77,8 +67,27 @@ const VisitDashboardItem = ( site: Site ) => {
 	);
 };
 
+const P2Badge = ( site: SiteData ) => {
```

Comment:
It's just an opinion, but I prefer to make the conditional where this element might or might not appear explicit. So here, instead of having the `if` inside the `P2Badge` element, I'd surround the element with the conditional, like that:

```js
{site.options && site.options.is_wpforteams_site && <SitesP2Badge>P2</SitesP2Badge>}
```

Reply:
Another small JS tip, you can use the `?` operator for this. So:
```js
site.options?.is_wpforteams_site // is the same as
// site.options && site.options.is_wpforteams_site
```

---

There's nothing wrong with writing helper functions for this sort of thing, so if it's clearer to you that's fine. But Luis' right that it's fairly idiomatic in our code to just inline this sort of logic directly within the main render function.

Reply:
Applied with 03c84dd237